tempesta_fw/http_sess.c

```diff
@@ -847,23 +852,30 @@ tfw_http_sess_remove(TfwHttpSess *sess)
  * Challenged client must not send request before challenging timeout passed.
  */
 static int
-tfw_http_sess_check_jsch(StickyVal *sv)
+tfw_http_sess_check_jsch(StickyVal *sv, TfwHttpReq* req)
 {
-	time_t cur_time, min_time, max_time;
+	unsigned long cur_time, min_time, max_time;
 
 	if (!tfw_cfg_js_ch)
 		return 0;
 
 	cur_time = jiffies;
 	min_time = sv->ts + tfw_cfg_js_ch->delay_min
-			+ sv->ts % tfw_cfg_js_ch->delay_range;
+			+ msecs_to_jiffies(sv->ts % tfw_cfg_js_ch->delay_range);
```

Comment:
I see, that this unusual timeout calculation is intended for consistency with calculations in `js_challenge.js.tpl`, but an explanatory comment would be appropriate here.